test/index.test.ts

```diff
@@ -413,16 +431,32 @@ describe('HipThrusTS', () => {
         const returnAssignableToCorrectShouldFail: ReturnAssignableToCorrect = false;
         // @ts-expect-error
         const returnAssignableFromCorrectShouldFail: ReturnAssignableFromCorrect = false;
+
+        const pipedAToBStringToBNumberAttachDataReturnValues = await pipedAToBStringToBNumber.attachData(
+          { a: aPassedIn }
+        );
+
+        expect(pipedAToBStringToBNumberAttachDataReturnValues).to.has.property(
+          'b'
+        );
+        expect(pipedAToBStringToBNumberAttachDataReturnValues.b).to.be.equal(5);
       });
-      it('attaches properly typed data from left sync data attacher and right not fully covered sync data attacher', () => {
+      it('attaches properly typed data from left sync data attacher and right not fully covered sync data attacher', async () => {
+        const aPassedIn = 'some string';
+        const bPassedIn = 4;
+        const otherPassedIn = 'other string';
+
         const left = {
           attachData(context: { a: string }) {
+            expect(context.a).to.be.equal(aPassedIn);
             return { b: 4 };
           },
```

Comment:
- [ ] same drill everywhere ... basically get rid of all the bare constants ;)

Reply:
done